src/fundus/scraping/html.py

```diff
@@ -47,73 +74,101 @@ def supported_file_formats(self) -> List[str]:
         return list(self.archive_mapping.keys())
 
 
+_http_regex: Pattern[str] = re.compile(r"https?://(?:[a-zA-Z]|\d|[$-_@.&+]|[!*(),]|%[\da-fA-F][\da-fA-F])+")
+
+
+def validate_url(url: str) -> bool:
+    return bool(re.match(_http_regex, url))
```

Comment:
My original motivation to use a custom regex over `validators` was o have as few dependencies as possible. Now, since we're mostly pretty much down the application stream, getting another dependency like `validators` doesn't seem that bad. I would suggest replacing this with `validators`.

Reply:
I agree. `validators` also seems like a long-lasting and tested library.